File: js_ts/variaveis/ativi1/situacao.ts
const aluno = String(prompt("nome"));
const nota1 = Number(prompt("nota 1"));
const nota2 = Number(prompt("nota 2"));

const media: number = (nota1 + nota2) / 2;
const aprovado: boolean = media >= 7.0

console.log("aluno de nome: " , aluno)
console.log("com média: " , media)
console.log("está aprovado? " , aprovado)

File: python/variaveis/atividades1/situacao.py
aluno = "wesllen"
aprovado = False

nota1 = float(input("nota 1: "))
nota2 = float(input("nota 2: "))

media = (nota1 + nota2)/2
aprovado =  media >= 7.0

print("aluno de nome:", aluno)
print("com média:", media)
print("está aprovado?", aprovado)
